components/renku_data_services/crc_migrations/versions/024b215f9a14_default_pool_access_control.py

```diff
@@ -0,0 +1,32 @@
+"""add default pool access control
+
+Revision ID: 024b215f9a14
+Revises: 3097ca05ce65
+Create Date: 2023-1--15 12:33:45.53902
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "024b215f9a14"
+down_revision = "3097ca05ce65"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "users",
+        sa.Column("no_default_access", sa.Boolean(), nullable=False, default=False),
```

Comment:
**issue (blocking):** You need to set `default=False, server_default=expression.false()` to properly create a default value at the database level, otherwise the default only works for newly created values, but doesn't add the value on existing columns (it's incredibly stupid that alembic doesn't do this automatically, it caused me quite a bit of headache)